hazelcast/protocol/codec/count_down_latch__await_codec.py

```diff
@@ -0,0 +1,28 @@
+from hazelcast.protocol.builtin import FixSizedTypesCodec
```

Comment:
I meant deleting the whole file `hazelcast/protocol/codec/count_down_latch__await_codec.py`, as this is unnecessary for this PR